### PR TITLE
antimicrox: update to 3.2.5

### PR DESCRIPTION
**Pull request by IcedQuinn**
#### Testing the changes
- I tested the changes in this PR: yes

#### Local build testing
- I built this PR locally for my native architecture, amd64-glibc

### CODE REVIEW

File: srcpkgs/antimicrox/template [hunk 1]
```diff
@@ -12,23 +12,7 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/AntiMicroX/antimicrox"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=53c6b0e6c8ea8c3a5527b56e1ad052eae983ba36abfc090769c6164bc5f342dd
-
-libantimicrox_package() {
```
Comment (classabbyamp) [inline srcpkgs/antimicrox/template:17]: why remove these subpackages?

Comment (IcedQuinn) [reply srcpkgs/antimicrox/template:17]: the cmake file no longer produces them

Comment (classabbyamp) [reply srcpkgs/antimicrox/template:17]: ok, then the symlinks need to be removed, and those subpackages need to be added to the `replaces` list in `removed-packages`

Comment (classabbyamp) [reply srcpkgs/antimicrox/template:17]: and also the shlib needs to be removed from `common/shlibs`

Comment (classabbyamp) [reply srcpkgs/antimicrox/template:17]: still need to `rm srcpkgs/libantimicrox srcpkgs/libantimicrox-devel` and add them to the commit

**Comment by classabbyamp** [comment]
musl fail is caused by not having `libexecinfo-devel`:
```sh
case "$XBPS_TARGET_MACHINE" in
	*-musl) makedepends+=" libexecinfo-devel"
esac
```

**Comment by classabbyamp** [comment]
and please add the email you're using to commit with to your github account

File: srcpkgs/removed-packages/template [hunk 2]
```diff
@@ -245,6 +245,8 @@ replaces="
  key-mon<=1.17_5
  laditools<=1.1.0_3
  libapp<=20140527_2
+ libantimicrox>=3.1.3_1
+ libantimicrox-devel>=3.1.3_1
  libco-devel<=20_1
```
Comment (classabbyamp) [inline srcpkgs/removed-packages/template:250]: these should be <= not >=

File: srcpkgs/antimicrox/template [hunk 3]
```diff
@@ -12,23 +12,11 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/AntiMicroX/antimicrox"
```
Comment (classabbyamp) [inline srcpkgs/antimicrox/template:13]:
```suggestion
homepage="https://github.com/AntiMicroX/antimicrox"
changelog="https://raw.githubusercontent.com/AntiMicroX/antimicrox/master/CHANGELOG.md"
```

File: srcpkgs/antimicrox/template [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'antimicrox'
 pkgname=antimicrox
-version=3.1.3
+version=3.2.5
```
Comment (classabbyamp) [inline srcpkgs/antimicrox/template:3]: 3.3.1 is out

File: srcpkgs/antimicrox/template [hunk 5]
```diff
@@ -12,23 +12,11 @@ maintainer="fosslinux <fosslinux@aussies.space>"
...
-
-libantimicrox-devel_package() {
-	depends="libantimicrox>=${version}_${revision}"
-	short_desc+=" - development files"
-	pkg_install() {
-		vmove usr/lib/*.so
-		vmove usr/include
-	}
-}
+case "$XBPS_TARGET_MACHINE" in
```
Comment (oreo639) [inline srcpkgs/antimicrox/template:17]: You could probably just use `XBPS_TARGET_LIBC`.

File: srcpkgs/antimicrox/template [hunk 6]
```diff
@@ -12,23 +12,11 @@ maintainer="fosslinux <fosslinux@aussies.space>"
...
+	*-musl) makedepends+=" libexecinfo-devel"
```
Comment (classabbyamp) [inline srcpkgs/antimicrox/template:18]:
```suggestion
	*-musl)
		makedepends+=" libexecinfo-devel"
		configure_args+=" -DCMAKE_EXE_LINKER_FLAGS=-lexecinfo" ;;
```
this should fix the linker error on musl

File: srcpkgs/antimicrox/template (hunk 5 above)
Comment (classabbyamp) [reply srcpkgs/antimicrox/template:18]: yes, true:
```
if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
inner stuff
fi
```
is better

**Comment by IcedQuinn** [comment]
looks like we're green?

**Comment by classabbyamp** [comment]
can you just make this change that oreo suggested? https://github.com/void-linux/void-packages/pull/40095#discussion_r1005956401

**Comment by IcedQuinn** [comment]
> can you just make this change that oreo suggested? [#40095 (comment)](https://github.com/void-linux/void-packages/pull/40095#discussion_r1005956401)

i'm not sure that was a significant enough detail to block a merge over, but okay.

**Pull request merged**